code/game/objects/items/weapons/twohanded.dm

```diff
@@ -511,4 +513,44 @@
 	return 0
 
 /obj/item/weapon/twohanded/vibro_weapon/update_icon()
-	icon_state = "hfrequency[wielded]"
\ No newline at end of file
+	icon_state = "hfrequency[wielded]"
+
+/*
+ * Bone Axe
+ */
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()
+	icon_state = "bone_axe[wielded]"
+
+/*
+ * Bone Spear
+ */
+/obj/item/weapon/twohanded/bonespear	//Blatant imitation of spear, but made out of bone. Not valid for explosive modification.
+	icon_state = "bone_spear0"
+	name = "bone spear"
+	desc = "A large sharpened bone on the end of a long stick. A standard of ancient tribal warfare."
```

Comment:
"A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."

This is a joke about the normal spears description btw.